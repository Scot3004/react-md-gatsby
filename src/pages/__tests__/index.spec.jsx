import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../index';

it('renders correctly', () => {
  const data = {
    "allMarkdownRemark": {
      "edges": [
        "post_edges"
      ]
    }
  }

  const tree = shallow(<Home data={data} />);
  expect(toJson(tree)).toMatchSnapshot();
});
