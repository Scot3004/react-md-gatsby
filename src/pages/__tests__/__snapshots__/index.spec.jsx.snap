// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="index-container"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Sergio Orozco
    </title>
    <link
      href="https://www.secorto.com"
      rel="canonical"
    />
  </HelmetWrapper>
  <SEO
    config={
      Object {
        "blogPostDir": "posts",
        "copyright": "Copyright © 2018. Sergio Orozco",
        "disqusShortname": "scot3004",
        "fixedFooter": false,
        "pathPrefix": "/",
        "postDefaultCategoryID": "Tech",
        "shortTitle": "SeCOrTo",
        "siteDescription": "Sitio web presentación de Sergio Orozco Torres",
        "siteFBAppID": "291685517914169",
        "siteGATrackingID": "UA-10920504-3",
        "siteLogo": "/logos/logo-1024.png",
        "siteRss": "/rss.xml",
        "siteTitle": "Sergio Orozco",
        "siteTitleAlt": "Sergio Orozco Torres website",
        "siteUrl": "https://www.secorto.com",
        "userAvatar": "/logos/logo-192.png",
        "userDescription": "Yeah, I like animals better than people sometimes... Especially dogs. Dogs are the best. Every time you come home, they act like they haven't seen you in a year. And the good thing about dogs... is they got different dogs for different people.",
        "userLinks": Array [
          Object {
            "iconClassName": "fa fa-github",
            "label": "GitHub",
            "url": "https://github.com/Scot3004",
          },
          Object {
            "iconClassName": "fa fa-facebook",
            "label": "Facebook",
            "url": "https://facebook.com/Scot3004",
          },
          Object {
            "iconClassName": "fa fa-twitter",
            "label": "Twitter",
            "url": "https://twitter.com/Scot3004",
          },
        ],
        "userLocation": "Barranquilla, Colombia",
        "userName": "Sergio Orozco",
        "userTwitter": "scot3004",
      }
    }
    postEdges={
      Array [
        "post_edges",
      ]
    }
  />
  <PostListing
    postEdges={
      Array [
        "post_edges",
      ]
    }
  />
</div>
`;
